### PR TITLE
OCT-1571: Add link to wallet address view on Etherscan

### PR DESCRIPTION
**Pull request by jmikolajczyk**
## Definition of Done

1. [ ] Acceptance criteria are met.
2. [ ] PR is manually tested before the merge by developer(s).
    - [ ] Happy path is manually checked.
3. [ ] PR is manually tested by QA when their assistance is required (1).
    - [ ] Octant Areas & Test Cases are checked for impact and updated if required (2).
4. [ ] Unit tests are added unless there is a reason to omit them.
5. [ ] Automated tests are added when required.
6. [ ] The code is merged.
7. [ ] Tech documentation is added / updated, reviewed and approved (including mandatory approval by a code owner, should such exist for changed files).
    - [ ] BE: Swagger documentation is updated.
8. [ ] When required by QA:
    - [ ] Deployed to the relevant environment.
    - [ ] Passed system tests.

---

(1) Developer(s) in coordination with QA decide whether it's required. For small tickets introducing small changes QA assistance is most probably not required.

(2) [Octant Areas & Test Cases](https://docs.google.com/spreadsheets/d/1cRe6dxuKJV3a4ZskAwWEPvrFkQm6rEfyUCYwLTYw_Cc).

### CODE REVIEW

File: client/src/components/shared/Layout/LayoutWallet/LayoutWallet.tsx [hunk 1]
```diff
@@ -84,7 +87,15 @@ const LayoutWallet: FC<LayoutWalletProps> = ({ onDisconnect }) => {
         title={t('wallet')}
         titleSuffix={
           address ? (
-            <div className={styles.titleSuffix}>{truncateEthAddress(address)}</div>
+            <Button
+              className={styles.address}
+              dataTest="LayoutWallet__Button--address"
+              href={`${networkConfig.etherscanAddress}/address/${address}`}
+              variant="link3"
```
Comment (aziolek) [inline client/src/components/shared/Layout/LayoutWallet/LayoutWallet.tsx:94]: `link` and `link5` variants already have `arrowTopRight` declared inside. They can't be reused?

Comment (jmikolajczyk) [reply client/src/components/shared/Layout/LayoutWallet/LayoutWallet.tsx]: Nice catch -> changed to `link`

**Review by aziolek — CHANGES_REQUESTED** [review]
CYs fail.

**Review by aziolek — COMMENTED** [review]
I think we can unify this with `link` variant we already have. I asked Matt about that [here](https://linear.app/golemfoundation/issue/OCT-1571/add-link-to-wallet-address-view-on-etherscan#comment-16ec2ae3).

**Comment by jmikolajczyk** [comment]
> Please unify the `link`, as accepted [here](https://linear.app/golemfoundation/issue/OCT-1571/add-link-to-wallet-address-view-on-etherscan#comment-98b23e07).

Done

**Review by aziolek — APPROVED** [review]

**Pull request merged**